config/config.go

```diff
@@ -958,7 +957,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		TimeoutPrevoteDelta:         500 * time.Millisecond,
 		TimeoutPrecommit:            1000 * time.Millisecond,
 		TimeoutPrecommitDelta:       500 * time.Millisecond,
-		TimeoutCommit:               1000 * time.Millisecond,
+		TargetHeightDuration:        3500 * time.Millisecond,
```

Comment:
[question] why isn't this 15 seconds?

```suggestion
		TargetHeightDuration:        15 * time.Second,
```

Reply:
it will break a bunch of tests, since they're testing assuming specific block times

Reply:
got it, no change needed. Resolving.